autopush/websocket.py

```diff
@@ -1215,6 +1215,8 @@ def process_register(self, data):
             d = self.deferToThread(self.conf.make_endpoint,
                                    self.ps.uaid, chid, data.get("key"))
         else:
+            if self.conf.disable_simplepush:
+                return self.bad_message("register", "No longer supported")
```

Comment:
How is it possible for a non-webpush client to connect and issue a register command if we drop them after hello?